packages/create-plugin/src/commands/generate.command.ts

```diff
@@ -19,9 +23,8 @@ import { updateGoSdkAndModules } from './generate/update-go-sdk-and-packages.js'
 import { CliArgs, TemplateData } from '../types.js';
 
 export const generate = async (argv: minimist.ParsedArgs) => {
-  const answers = await promptUser(argv);
-  const templateData = getTemplateData(answers);
-  const exportPath = getExportPath(answers.pluginName, answers.orgName, answers.pluginType);
+  const templateData = getTemplateData(buildProxyFromUserPrompt(await promptUser(argv)));
```

Comment:
nit: it might just be me but I find we lose legibility by nesting multiple function calls inside each other. I have to read "from the inside out" rather than top to bottom which I think is more natural.